sdk/communication/azure-communication-administration/README.md

```diff
@@ -24,9 +24,18 @@ pip install azure-communication-administration
 # You can find your endpoint and access token from your resource in the Azure Portal
 import os
 from azure.communication.administration import PhoneNumberAdministrationClient
+from azure.identity import DefaultAzureCredential
 
 connection_str = os.getenv('AZURE_COMMUNICATION_SERVICE_CONNECTION_STRING')
+endpoint = os.getenv('AZURE_COMMUNICATION_SERVICE_ENDPOINT')
+
+# To use Azure Active Directory Authentication (DefaultAzureCredential) make sure to have
+# AZURE_TENANT_ID, AZURE_CLIENT_ID and AZURE_CLIENT_SECRET as env variables.
+phone_number_administration_client = PhoneNumberAdministrationClient(endpoint, DefaultAzureCredential())
+
+# You can also authenticate using your connection string
 phone_number_administration_client = PhoneNumberAdministrationClient.from_connection_string(connection_str)
```

Comment:
Do we need to remove this line (line 37), when there is an addition of line 34?

Reply:
We're keeping both forms of authentication, so line 37 is still relevant

Reply:
True, but it is quite weird to look at it in the code snippet so maybe keep only one in the first snippet.. then write a regular text saying that you could also authenticate using the connection string and then have a separate snippet containing just the separate constructor.. that way it would easier to look at and understand better.